src/UserInterface/templates/Number_Guesser.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Number Guesser</title>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            text-align: center;
+            padding: 20px;
+        }
+        .hidden {
+            display: none;
+        }
+        #message {
+            margin: 20px 0;
+            font-size: 1.2em;
+            color: #333;
+        }
+        input, button {
+            padding: 10px;
+            margin: 10px;
+            font-size: 1em;
+        }
+    </style>
+</head>
+<body>
+    <h1>Number Guesser</h1>
+    <div id="join">
+        <input type="text" id="player_id" placeholder="Enter Player ID">
+        <button onclick="joinGame()">Join Game</button>
+    </div>
```

Comment:
You should not need to/be able to enter a custom player id. The uuids are automatically generated and saved as cookies,